### DISCUSSION TITLE
Dose csdm can run on postgres version 9.4.26 ?

### DISCUSSION THREAD

**Discussion by HeMuxD** [akiver/cs-demo-manager#695] [Q&A] [answered]
My computer cant install postgres 16.1 due to compatibility issues, can it run on lower versions postgres? For example 9.4.26.
![1](https://github.com/akiver/cs-demo-manager/assets/130919860/c57edb9f-6889-4699-ba0b-a45836e3dd2e)
This is the error when I install postgres 16.1.
![2](https://github.com/akiver/cs-demo-manager/assets/130919860/376e5ebf-322a-41c3-bb0f-f3fcab6a52d3)
This is the error when i use postgres 9.4.26 on csdm, its say "syntax error in generated"

**Comment by akiver** [comment, marked as answer]
I doubt the version 9 would work - it's too old.
I think you should fix the Postgres installation issue instead of trying to install an old version.

A quick search lead to user/rights issue:

https://superuser.com/questions/1475822/postgresql-install-fail-in-windows-10-icacls-error-access-denied
https://stackoverflow.com/questions/55282696/issues-while-installing-postgresql-in-windows-10
